src/App/Gallery/Gallery-Image/Gallery-Image.style.js

```diff
@@ -12,7 +12,7 @@ export const Container = styled.div`
   }
   p {
     font-family: lato, sans-serif;
-    font-size: 9px;
+    font-size: 13px;
```

Comment:
Any downside to using em for font size?

Reply:
amazon uses px's so its easier to copy this way